docs/UserGuide.md

```diff
@@ -230,8 +230,8 @@ The following points explain the format of a command.
 
 ### Company Restrictions <a id="c-company-restrictions"></a>
 
-* The `COMPANY_NAME` should only contain alphanumeric characters and spaces, and it should not be blank.
-* The `PHONE_NUMBER` should only contain numbers, and it should be at least 3 digits long.
+* The `COMPANY_NAME` should only contain alphanumeric characters and spaces, but not blank, and have a character limit of at most 30.
```

Comment:
I am just thinking if the "not be blank"  should be in command restrictions. Technically, command restrictions are meant to be applied throughout the UG. So do you think will confuse the user when it comes to edit company/role?

Reply:
The reason why I left it in was that spaces are allowed hence some people can argue that by placing in a few spaces they did not leave the input empty. As for the edit command, the input of "" and " " will still display N/A in the UI, hence not necessary.

Reply:
After further discussion, changes have been made with a new commit.